lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
I needed to squash the migrations in order to add `course_id` to `VisibleBlocks` for bulk-reads, without providing a nonsensical default for that column.